core/src/apps/webauthn/fido2.py

```diff
@@ -1510,8 +1515,14 @@ def cbor_make_credential_sign(
         + extensions
     )
 
+    app = knownapps.by_rp_id_hash(cred.rp_id_hash)
```

Comment:
we could move `_DEFAULT_USE_SIGN_COUNT` and `_DEFAULT_USE_SELF_ATTESTATION` to `knownapps.py`, prefill the fiels there, and simplify this code to:
```
if app is not None:
	use_self_attestation = app.use_self_attestation
```

Reply:
I actually had the same thought, but the code wouldn't really simplify, because you still need to have the `else` branch:
```
    if app is not None:
        use_self_attestation = app.use_self_attestation
    else:
        use_self_attestation = _DEFAULT_USE_SELF_ATTESTATION
```
In order to simplify this you'd also need `knownapps.by_rp_id_hash()` to always return a FIDOApp object, and when it doesn't find it by RP ID, return a "default" FIDOApp. But then it's questionable, what label the default app should have.